### PR TITLE
Task #153167 feat: To add ACL for TJNotification

### PR DESCRIPTION
**Pull request by Mayaank12**

### CODE REVIEW

File: src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini [hunk 1]
```diff
@@ -93,3 +93,11 @@ COM_TJNOTIFICATIONS_CORE_TEMPLATE_VALUE="No"
 COM_TJNOTIFICATIONS_TAGS_DESC="Enter the tag given on left in to the editor these tag names will be replaced by description given in notification template"
 COM_TJNOTIFICATIONS_REPLACEMENT_TAGS="Replacement Tag"
 COM_TJNOTIFICATIONS_REPLACEMENT_TAGS_DESC="Description"
+
+; Since
```
Comment (manojLondhe) [inline src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini]: Since `__DEPLOY_VERSION__`

Comment (Mayaank12) [reply src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini]: Updated

**Comment by manojLondhe** [comment]
@snehal0904 @Mayaank12 will migration will be needed for this?

**Comment by manojLondhe** [comment]
@snehal0904 @Mayaank12 will migration will be needed for this?

**Comment by Mayaank120495** [comment]
@manojLondhe No sir

**Pull request merged**